backend/app/services/knowledge/permission_request_service.py

```
@@ -0,0 +1,627 @@
+        Raises:
+            ValueError: If validation fails or permission denied
+        """
+        # Get the request
+        request = (
+            db.query(PermissionRequest)
+            .filter(PermissionRequest.id == request_id)
+            .first()
+        )
+
+        if not request:
+            raise ValueError("Permission request not found")
```

Ruff (0.14.14)

Avoid specifying long messages outside the exception class

(TRY003)